yt_dlp/extractor/kanal2.py

```diff
@@ -0,0 +1,122 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from datetime import datetime
+import re
+import time
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    url_or_none,
+    traverse_obj,
+)
+
+
+class Kanal2IE(InfoExtractor):
+    SUBTITLE_DATE_RE = re.compile(r'\((\d{2}\.\d{2}\.\d{4}\s\d{2}:\d{2})\)$')
+
+    _VALID_URL = r'https?://.+\.postimees\.ee/[^?]+\?(.*?&)?id=(?P<id>\d+)'
```

Comment:
will this extractor work with other postimees.ee sites? if so, then should it still be named Kanal2?
```suggestion
    _VALID_URL = r'https?://kanal2\.postimees\.ee/[^?]+\?(.*?&)?id=(?P<id>\d+)'
```
or if we do want to match any subdomain then we should do it safely:
```suggestion
    _VALID_URL = r'https?://(?:[\w\d-]+\.)?postimees\.ee/[^?]+\?(.*?&)?id=(?P<id>\d+)'
```